packages/core-data/src/actions.js

```diff
@@ -56,13 +61,13 @@ export function addEntities( entities ) {
  * @param {string}       kind    Kind of the received entity.
  * @param {string}       name    Name of the received entity.
  * @param {Array|Object} records Records received.
+ * @param {Object?}      query  Query Object.
```

Comment:
Based on documentation, nullable type should be denoted with `?` as _prefix_, not _suffix_:

http://usejsdoc.org/tags-type.html